app/api/schema/events.py

```diff
@@ -83,6 +83,7 @@ def validate_timezone(self, data, original_data):
     is_billing_info_mandatory = fields.Bool(default=False)
     is_donation_enabled = fields.Bool(default=False)
     is_chat_enabled = fields.Bool(default=False)
+    chat_room_id = fields.Str(allow_none=True)
```

Comment:
Shouldn't be needed, this is internal and not editable or viewable

Reply:
room id is needed to add a user into a room. From where will it be accessed?

Reply:
Is it needed in frontend?

Reply:
If so, this should be dumps_only

Reply:
not in frontend, but for add user api call